course/layouts/partials/topic.html

```diff
@@ -1,9 +1,12 @@
 {{ $index := .index }}
 {{ $context := .context }}
+{{ $subtopics := slice }}
+{{ if not (eq .subtopics nil) }}
 {{ $subtopics := .subtopics }}
```

Comment:
You need to change this to a `=`, otherwise it will create a variable which goes out of scope in the next line